toolset/setup/linux/languages/ringojs.sh

```diff
@@ -1,12 +1,14 @@
 #!/bin/bash
 
-RETCODE=$(fw_exists ${IROOT}/ringojs_0.10.installed)
+RETCODE=$(fw_exists ${IROOT}/ringojs_0.11.installed)
 [ ! "$RETCODE" == 0 ] || { return 0; }
 
-fw_get http://www.ringojs.org/downloads/ringojs_0.10-1_all.deb
+fw_get https://github.com/ringo/ringojs/releases/download/v0.11.0/ringojs_0.11_all.deb -O ringojs_0.11_all.deb
 sudo apt-get install -y jsvc
-sudo dpkg -i ringojs_0.10-1_all.deb
+sudo dpkg -i ringojs_0.11_all.deb
```

Comment:
FYI, the final goal is to remove sudo from these scripts as well. It would be much better to install ringo into a prefix folder (such as `$IROOT/ringojs`) and completely avoid modifying the base OS image. If you can add that to this PR it would be a huge step in teh right direction